README.md

```diff
@@ -126,7 +130,7 @@ class App extends Singleton
 }
 ```
 
-This facade might be used everywhere you need using `firestarter` function. Example: `firestarter()->posts()->doSth()`.
+This facade might be used everywhere you need using `fs` function. Example: `fs()->posts()->doSth()`.
```

Comment:
That's something to discuss. I've changed this because it is faster to write `fs` than `firestarter` in the core files and it might be useful when we will use this more times. But maybe let's get back to `firestarter` to avoid confusion to `fs` (filesystem) and create snippets to write this faster :D I'll change this back :)

Reply:
I've changed function names back.